src/dbnode/server/server.go

```diff
@@ -410,27 +408,30 @@ func Run(runOpts RunOptions) {
 	defer stopReporting()
 
 	// Setup query stats tracking.
-	statsOpts := stats.QueryStatsOptions{
+	docsLimit := limits.QueryLimitOptions{
```

Comment:
nit: can we remove the concept of `DefaultLookback` and require it to be explicitly set? Then we can have limits apply only iff `lookback > 0` and `limit > 0`

Reply:
Same comment. We want default lookback so we get metrics even without limits.

Reply:
Ah that makes sense; can we hide it behind a `limits.NewQueryLimitOptions()` method then?

Reply:
Done